### ISSUE TITLE
Cannot display contributors without images on the website

### ISSUE THREAD

**Issue by gpapp** [podlove/podlove-publisher#1398]
Minor issue, 3.x-4.x versions contributors plugin when displaying contirbutors without uploaded images

e.g. using the following snippet 
[podlove-episode-contributor-list preset="table" group="visitor" donations="no"]

### Expected behavior
When displaying the list of avatars on the website, from a twig as provided in the website examples, the contributors without images set should be displayed with the inline image/svg+xml:... image. It works well within the contributor editing screen.

### Actual behavior
The generated URL for the avatars contains the http base as a prefix. 

```
https://xxxx.net/EPISODE_NAME/image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGNsYXNzPSJoLTYgdy02IiBmaWxsPSJub25lIiB2aWV3Qm94PSIwIDAgMjQgMjQiIHN0cm9rZT0iY3VycmVudENvbG9yIj4KICA8cGF0aCBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIHN0cm9rZS13aWR0aD0iMiIgZD0iTTUuMTIxIDE3LjgwNEExMy45MzcgMTMuOTM3IDAgMDExMiAxNmMyLjUgMCA0Ljg0Ny42NTUgNi44NzkgMS44MDRNMTUgMTBhMyAzIDAgMTEtNiAwIDMgMyAwIDAxNiAwem02IDJhOSA5IDAgMTEtMTggMCA5IDkgMCAwMTE4IDB6IiAvPgo8L3N2Zz4K
```

### System information (see `Podlove > Support` menu)
```
Website                    https://zartosztaly.hu
PHP Version                8.1.12-1ubuntu4.3
WordPress Version          6.3.2
WordPress Theme            Zártosztály Podcast v2 v1.0
Active Plugins             
           - Akismet Anti-spam: Spam Protection v5.3
           - Site Kit by Google v1.111.1
           - Jetpack Protect v1.4.2
           - Jetpack v12.7.1
           - Limit Login Attempts Reloaded v2.25.26
           - Podlove Podcast Publisher v4.0.1
           - Podlove Web Player v5.7.1
           - Social Media Auto Publish v3.2
           - Converter for Media v5.11.2
           - Pexels: Free Stock Photos v1.2.2
WordPress Database Charset utf8
WordPress Database Collate 
Publisher Version          4.0.1
Publisher Database Version 153
Web Player Version         player_v5
open_basedir               ok
curl Version               7.88.1
iconv                      available
simplexml                  ok
max_execution_time         30
upload_max_filesize        32M
memory_limit               256M
disable_classes            
disable_functions          
permalinks                 ok (/%postname%/)
podlove_permalinks         ok
podcast_settings           ok
web_player                 ok
podlove_cache              on
assets                     
  - mp3    audio/mpeg       https://zartosztaly.hu/feed/mp3/
  - mp4    video/mp4        https://zartosztaly.hu/feed/mp4/
  - jpg    image/jpeg       no feed
  - jpg    image/jpeg       no feed
  - psc    application/xml  no feed
cron                       ok
duplicate_guids            ok

0 errors
0 notices
Nice, Everything looks fine!
```

**Comment by dnkbln** [comment]
Hello Gergely, thank you for your feedback.

I try to reproduce your issue. If I use the group option in the podlove-episode-contributor-list template, no list of contributors is displayed on the episode page. If I omit the option, the list is displayed correctly, even if no image is stored for the contributor. 

Can you confirm this?

**Comment by dnkbln** [comment]
Do you have more then one group? Is the visitor-group the first group?

If you use the first group in the list for a contributor, in some case the group is not set. Workaround select an other group and set the group back.

**Comment by gpapp** [comment]
I can confirm, that happens, but that's a different bug than originally reported. That is about not displaying the icon properly

Website:
![image](https://github.com/podlove/podlove-publisher/assets/4165192/14056581-f224-44ff-a06f-f706e2ae024c)
Admin:
![image](https://github.com/podlove/podlove-publisher/assets/4165192/3889602e-710f-481d-a0fa-4e0e24795925)

That is just to do with displaying the embedded SVG properly, this is another I didn't notice yet.

For the other issue, the groups for the visitor contributors seem, that they are not saved starting from the last update. The older episodes that had contributors show up. Changing the group of the visitor to presenter does not show either and changing them back does not solve it.

**Comment by eteubert** [comment]
> False alarm. Somehow I managed to turn visibility off, when creating new contributor.

Is this issue solved then?

**Comment by gpapp** [comment]
Just checked. Resolved as of 4.0.3
